container/ubuntu16_04/layers/bazel/bazel_tools.yaml

```diff
@@ -1,6 +1,11 @@
 schemaVersion: "2.0.0"
```

Comment:
Can we add a check for execution of "bazel version" in one of these yaml files?

Reply:
Done.